packages/kolibri-tools/jest.conf/setup.js

```diff
@@ -42,6 +45,13 @@ Vue.config.silent = true;
 Vue.config.devtools = false;
 Vue.config.productionTip = false;
 
+plugin_data['languageGlobals'] = {
+  languageCode: defaultLanguage.id,
+  languages: {
+    defaultLanguage: defaultLanguage,
```

Comment:
I feel like this should be `defaultLanguage.id` as the `languages` object is meant to be a lookup from the language code to the language object?